src/org/folio/models/EurekaTenant.groovy

```diff
@@ -0,0 +1,39 @@
+package org.folio.models
+
+/**
+ * EurekaTenant class representing a tenant configuration for Eureka.
+ * It provides chainable setter methods following builder pattern for ease of use.
+ */
+class EurekaTenant extends Tenant {
+  /**
+   * Keycloak client identifier.
+   * Deafault: 'sidecar-module-access-client'
+   */
+  String clientId
+
+  /**
+   * Some secret phrase to authenticate in Keycloak.
+   * Is stored in AWS SSM Parameter Store or Hashicorp Vault.
+   */
+  String clientSecret
+
+  /**
+   * Keycloak service URL.
+   * Is the same for all Tenants
+   */
+  static String keycloakUrl
```

Comment:
Not sure thath this field should be static, as you will not be able to assign value to it